spec/models/fae/role_spec.rb

```diff
@@ -34,4 +34,12 @@
     end
   end
 
+  describe 'roles#create' do
```

Comment:
@andrewdang17 Love the addition of the test, however it's failing on Jenkins in Rails 4.1. Was this passing for you with appraisals?

```
06:00:21   1) Fae::Role default_scope should order by position
06:00:21      Failure/Error: expect(Fae::Role.all).to eq([role1, role2, role3])
06:00:21        
06:00:21        expected: [#<Fae::Role id: 3, name: nil, position: 1, created_at: "2017-06-04 05:59:09", updated_at: "2017-06-04 05:59:09">, #<Fae::Role id: 4, name: nil, position: 1, created_at: "2017-06-04 05:59:09", updated_at: "2017-06-04 05:59:09">, #<Fae::Role id: 2, name: nil, position: 2, created_at: "2017-06-04 05:59:09", updated_at: "2017-06-04 05:59:09">]
06:00:21             got: #<ActiveRecord::Relation [#<Fae::Role id: 4, name: nil, position: 1, created_at: "2017-06-04 05:59:09", updated_at: "2017-06-04 05:59:09">, #<Fae::Role id: 3, name: nil, position: 2, created_at: "2017-06-04 05:59:09", updated_at: "2017-06-04 05:59:09">, #<Fae::Role id: 2, name: nil, position: 4, created_at: "2017-06-04 05:59:09", updated_at: "2017-06-04 05:59:09">]>
```